### PR TITLE
Removes security equipment from all cyborg modules except security

### PR DESCRIPTION
**Pull request by zaracka**
:cl:
balance: Non-security cyborgs no longer have security equipment. No more flashes. No more zipties.
/:cl:

The main reason for not doing this seems to be the fact that they've always had flashes. If there's an objective reason they need to have flashes please comment it below. I feel cyborgs without security equipment would be more dedicated to their specialties determined by their module selection. This is less so a nerf and more so a redirection of playstyle.

### CODE REVIEW

**Comment by davethwave** [comment]
the flashes are their eyes!

**Comment by evsey9** [comment]
what davethwave said, also this makes restraining people and non-humans be possible only by beating their head in until they fall into deep crit

**Comment by evsey9** [comment]
also this is a direct nerf because secborgs are disabled forever

**Comment by zaracka** [comment]
that's true, but I think the whole point of cyborgs is they're second class so they shouldn't be restraining humans, rather running away in the case of self-defense. they shouldn't really be restraining non-humans on behalf of humans, that's not their job. they can however attack non-humans who are harassing them without issue. the lore part seems to be a problem, but it's not particularly rigid and the idea that flashes are disabled for optical capability isn't farfetched (the real problem is flashes providing vision, but that's just how it is)

**Comment by zaracka** [comment]
one of the comments I received in OOC was it would turn a situation where a mediborg is flash-spamming a traitor to sec into a situation where they would tend to the injured instead. same idea would apply to engiborgs fixing the SM etc.

however feel free to close this if you disagree from your maintainer perspective

**Comment by Kagus** [comment]
Cyborgs are intelligent, independently mobile machines. They do whatever their laws and programming dictate that they do. Not all silicons are bound by Asimov laws, and the ones that are won't necessarily remain as such. Cyborgs are tools, and removing their ability to _non-lethally_ subdue someone they through whatever laws and situation are targeting just means that situations are going to be handled in a more dangerous, potentially lethal fashion.

And you're removing zipties from the _syndicate_ cyborg as well, which I really don't understand... For whatever lore or fluff reason you have about removing stun capabilities from non-emagged borgs, _surely_ the syndicate isn't beholden to those regulations?

**Comment by Wilchenx** [comment]
Is this a salt pr?

**Comment by Kagus** [comment]
I think it's more likely just a misguided/misunderstood PR than salt. I can see where the argument about removing aggressive options from a borg would make it focus on other maintenance tasks instead is thinking, but I can also see where that thinking is just wrong from a gameplay-in-practice point of view.

**Comment by zaracka** [comment]
I removed zipties from the standard module. the reason I removed flashes from syndiborgs was because I figured they were a Nanotrasen security thing, meanwhile syndiborgs have syndicate weaponry. more over I wanted consistency with secborgs being removed. there's nothing to say you can't give mediborgs morphine, but this pr is to bring cyborgs inline with 'don't be security' sentiment by code rather than policy.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -540,7 +531,6 @@
 /obj/item/robot_module/syndicate
 	name = "Syndicate Assault"
 	basic_modules = list(
-		/obj/item/assembly/flash/cyborg,
```
Comment (Wilchenx) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Why did you remove flashes from syndicate cyborgs

**Comment by Wilchenx** [comment]
"but this pr is to bring cyborgs inline with 'don't be security' sentiment by code rather than policy."

Cyborgs are free to follow space law at will,

image

What are you reffering to by saying "rather than policy"?

**Comment by RogueSteampunker** [comment]
ITT: someone tried to reason that rule one is possible to follow without any means of stopping harm without causing harm

Like, come on. There is absolutely no reason that we should force borgs to be even more useless in situations of danger, especially when their laws basically route them towards needing to stop harm before preventing it. Hence, the idea of a borg running off to heal people or fix the SM when someone is cutting people down right in front of them doesn't even fucking work

**Comment by zaracka** [comment]
wilchen, the sentiment exists, I alleviate it with code instead of policy discussion

roguesteampunker, they can stop harm without sec equipment, each module does it differently

**Comment by Wilchenx** [comment]
>roguesteampunker, they can stop harm without sec equipment, each module does it differently

How can clown/janitor/service module stop harm without a flash?

>wilchen, the sentiment exists, I alleviate it with code instead of policy discussion

image

🤔

**Comment by zaracka** [comment]
pull them away

**Comment by RogueSteampunker** [comment]
>roguesteampunker, they can stop harm without sec equipment, each module does it differently

How even? last I checked, medbots didn't get morphine, engineers are useless, standard's soap is for cleaning and nothing else, and janitors only get slips when they're emagged. The only case to be made is service, and they need to get morphine from med to even make their stun drinks

**Comment by RogueSteampunker** [comment]
And pulling away people works for the fucking half second before they immediately go back at them. Seriously, all you're advocating for is borg suicide.

**Comment by Cyberboss** [comment]
>Implying sec borgs will ever be a thing

Yeah, let's not nerf silicons out of existence please

**Comment by zaracka** [comment]
pull away the guy who's being harmed, not the one attacking. if they're both fighting it's either voluntary or you shouldn't interfere cause interfering will give one the upperhand leading to harm which is breaking Asimov.

cyberboss, I don't think I implied that but i'm fine with your decision regardless

**Comment by Iamgoofball** [comment]
>secborgs are re-enabled

worst head admins recall when

**Comment by optimumtact** [comment]
goof this was ages ago calm yourself
